project_resource_calendar/views/room_type_view.xml

```diff
@@ -6,11 +6,14 @@
         <field name="view_type">form</field>
         <field name="view_mode">tree,form</field>
     </record>
+    <menuitem id="configuration_sub_menu"
```

Comment:
The last menu in the calendar root menu is  Reports it has a sequence of 300. In order to put the Configuration menu after we can use sequence=310. So later if we want to add new menus between them we can add sequence 301,302,303,304,...309 (so i think better to add sequence=310)

Reply:
@rimbendhaou Changed